src/reducers/targets.js

```diff
@@ -23,6 +26,11 @@ const reducer = function (state, action) {
                 .filter(target => target.isStage)[0] || {},
             editingTarget: action.editingTarget
         });
+    case HIGHLIGHT_TARGET:
+        return Object.assign({}, state, {
+            highlightedTargetId: action.targetId,
+            highlightedTargetTime: action.updateTime
```

Comment:
Seems like you could just put `Date.now()` here and not have to know about that detail when you dispatch.

Reply:
I think that would break the backwards/forwards-ness of redux though? I can put it in the action creator though, that would be safe.